tests/robot-tests/tests/libs/common.robot

```diff
@@ -773,13 +773,9 @@ user closes Set Page View box
     user waits until element is not visible    id:editingMode
 
 user checks page for details section
-    [Arguments]    ${heading}
-    user checks page contains element    testid:Expand Details Section ${heading}
-
-user expands details section
-    [Arguments]    ${heading}
-    user clicks element    testid:Expand Details Section ${heading}
+    [Arguments]    ${heading}    ${wait}=${timeout}
+    user waits until page contains element    testid:Expand Details Section ${heading}    ${wait}
 
 user checks page for details dropdown
```

Comment:
This keyword can also be renamed to `user waits until...`

But I also think this can be removed potentially?